src/lib/containers/discussion_forum/ForumTable.tsx

```diff
@@ -55,6 +108,14 @@ export const ForumTable: React.FC<ForumTableProps> = ({
 
   return (
     <div className="ForumTable bootstrap-4-backport">
+      <div className="ForumTable__top-level-control">
+        <Button
+          variant={subscribed ? 'outline-primary' : 'primary'}
+          onClick={() => handleFollowBtn()}
```

Comment:
Can you make the button `disabled` while the post/delete is in progress? It would prevent a whole class of potential race conditions, and also gives feedback that the request is ongoing.